openpype/hosts/maya/plugins/publish/collect_skeleton_mesh.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+from maya import cmds  # noqa
+import pyblish.api
+
+
+class CollectSkeletonMesh(pyblish.api.InstancePlugin):
+    """Collect Static Rig Data for FBX Extractor."""
+
+    order = pyblish.api.CollectorOrder + 0.2
+    label = "Collect Skeleton Mesh"
+    hosts = ["maya"]
+    families = ["rig"]
+
+    def process(self, instance):
+        frame = cmds.currentTime(query=True)
+        instance.data["frameStart"] = frame
+        instance.data["frameEnd"] = frame
+        skeleton_sets = [
+            i for i in instance
+            if i.lower().endswith("skeletonanim_set")
+        ]
+
+        skeleton_mesh_sets = [
+            i for i in instance
+            if i.lower().endswith("skeletonmesh_set")
+        ]
+        if not skeleton_sets and skeleton_mesh_sets:
+            self.log.debug(
+                "no skeleton_set or skeleton_mesh set was found....")
+            return
```

Comment:
Are we sure this logic is correct? This returns only when there are NO skeleton sets but there ARE skeleton mesh sets? Pretty sure this is wrong.